Makefile.menhir

```diff
@@ -27,6 +27,11 @@ MENHIRFLAGS := --explain --ocamlc "$(CAMLC) $(COMPFLAGS)" --infer\
 # (which is used in polymorphic variant), but is not currently used by
 # the grammar.
 
+# --no-stdlib means that we do *not* rely on Menhir's standard library.
+# Instead, if needed, we copy definitions from Menhir's library into our
+# parsers. This makes our parsers more robust in the face of changes
+# (additions/deletions) in Menhir's standard library.
+
```

Comment:
Isn't `--no-stdlib` redundant with the `import-menhirLib` target then?

Reply:
Apparently not, `menhirLib` is different from `standard.mly`, nevermind then :)

Reply:
I don't think so: what `import-menhirLib` does is to copy the menhir *runtime*, not the standard library. Even if we did use the standard library (without `--no-stdlib`), we wouldn't need to include it in the bootstrap as it is only used in the parser compilation phase, and we already store the result of this phase as `parser.ml`.

@fpottier: on the other end, I'm not sure why we should use `--no-stdlib`. It sounds like you don't want to deal with backward-compatibility issues when you evolve Menhir, so you encourage all users to copy/paste definitions from your standard library instead of actually using it. But then why have it at all? I think we could use the standard library directly, to have a shorter, nicer grammar, and then of course we would complain if you break stuff across versions -- as all other users would, so it's not bad if you hear the complaints.

Reply:
Menhir's standard library is for definitions of nonterminal symbols, like lists, separated lists, etc. Whereas MenhirLib is Menhir's runtime support library.